index.html

```diff
@@ -2191,60 +2164,18 @@ <h3>The <dfn>NDEFWriteOptions</dfn> dictionary</h3>
   <section data-dfn-for="NDEFScanOptions">
     <h3>The <dfn>NDEFScanOptions</dfn> dictionary</h3>
       <p>
-        To describe which messages an application is interested in, the
+        To pass an [= AbortSignal/aborted flag =], the
```

Comment:
Shall we express the intent of the aborted flag instead?
```suggestion
        To stop listening to NDEF messages, the

```

Reply:
OK! Thanks

Reply:
What about this?
```
        To stop listening to NDEF messages, an [= AbortSignal/aborted flag =]
        can be passed to the <a>NDEFScanOptions</a> dictionary:
```